liveblog.php

```diff
@@ -236,11 +238,19 @@ private static function register_embed_handlers() {
 	 */
 	public static function init() {
 		/**
-		 * Add liveblog support to the 'post' post type. This is done here so
-		 * we can possibly introduce this to other post types later.
+		 * Add liveblog support to the 'post' and 'page' post type.
+		 */		
+		$post_types = array( 'post', 'page' );
```

Comment:
If I understand correctly, this will also add support for pages out of the box? I'd rather we just stick to posts, and allowing pages to be added using the filter.